### ISSUE TITLE
[Merge]MONDO:0019847 congenital adrenal hypoplasia of maternal cause with MONDO_0010264 X-linked adrenal hypoplasia congenita

### ISSUE THREAD

**Issue by lcdaugherty** [monarch-initiative/mondo#1840]
**Mondo term (ID and Label)**
http://purl.obolibrary.org/obo/MONDO_0019847 with http://purl.obolibrary.org/obo/MONDO_0010264

**Reason for deprecation**
MONDO:0019847 congenital adrenal hypoplasia of maternal cause-  Property value is exactMatch Orphanet:95701 which does not exist. Maybe caused by a typo as I think it should be Orphanet:9570**2**

This might have been caused by MeSH: C536757 (which is now obsolete) but I am not sure. I cannot figure out what now should be the MeSH term for MONDO_0010264, any suggestions?

**Term to be merged with**
MONDO_0010264 X-linked adrenal hypoplasia congenita

**Comment by nicolevasilevsky** [comment]
This may be the correct MESH term:
https://meshb.nlm.nih.gov/record/ui?ui=D000075262

**Comment by nicolevasilevsky** [comment]
@lcdaugherty I'm not sure that MONDO:0019847 congenital adrenal hypoplasia of maternal cause is equivalent to MONDO_0010264 X-linked adrenal hypoplasia congenita.

When I tried to merge the two, there were issues when I ran the reasoner because 'congenital adrenal hypoplasia of maternal cause' is classified as a child of 'acquired chronic primary adrenal insufficiency' (according to Orphanet).

Maybe it is best to just obsolete this class and add a 'consider' to MONDO_0010264 X-linked adrenal hypoplasia congenita.
